_pages/software-tools/zoom.md

```diff
@@ -7,7 +7,7 @@ GSA offers [Zoom for Government](https://zoomgov.com/).
 ## Setup
 
 1. Install the Zoom application through the Self Service app.
-1. When signing in, select `.zoomgov.com`.
+1. When signing in, select `gsa.zoomgov.com`.
```

Comment:
It's referring to this:

<img width="316" alt="Screen Shot 2020-07-22 at 1 09 13 PM" src="https://user-images.githubusercontent.com/86842/88206647-976ee800-cc1c-11ea-9333-d0139283f6e3.png">

Perhaps it's just missing a step after `Sign in with SSO` to enter `gsa` for the `company domain`?

Reply:
Hmm, I have Zoom 5.0.2 and I'm not seeing that dropdown menu within the "Sign In" dialog box.

![Screen Shot 2020-07-22 at 1 38 57 PM](https://user-images.githubusercontent.com/443389/88215174-def77300-cc20-11ea-96fc-177cc14cf289.png)

In fact, I had to go to "Sign In with SSO" and select "I don't know the company domain" and enter my email address because I couldn't figure out how to select `.zoomgov.com` instead of `.zoom.us`.

Maybe something changed in their UI? Or maybe I installed the Zoom app differently?

Reply:
I don't see that option from the drop down either. @afeld have you upgraded your zoom in a while? I suggest doing that and we can open up a new pull request to address that step if it is still needed.